packages/edit-site/src/components/post-edit/index.js

```diff
@@ -50,11 +68,20 @@ function PostEditForm( { postType, postId } ) {
 					'getEditedEntityRecord',
 					args
 				),
+				records:
+					ids.length > 1
+						? ids.map( ( id ) =>
+								select( coreDataStore ).getEditedEntityRecord(
+									'postType',
+									postType,
+									id
+								)
+						  )
+						: null,
```

Comment:
I will replace this with `getEditedEntityRecords` once https://github.com/WordPress/gutenberg/pull/67872 is merged.